util/find_missing_timeline_translations.js

```diff
@@ -1,9 +1,12 @@
 import fs from 'fs';
 import path from 'path';
 import Regexes from '../resources/regexes';
+import NetRegexes from '../resources/netregexes';
 import { Timeline } from '../ui/raidboss/timeline';
 import { commonReplacement, partialCommonReplacementKeys } from '../ui/raidboss/common_replacement';
 
+NetRegexes.setFlagTranslationsNeeded(true);
```

Comment:
Sorry to be a pain, but should this have an additional comment here about why we're turning this on in the test file? It's kind of obvious when looking at files in the reverse manner (code -> test), but it might not be immediately obvious when looking at the test file alone.

Reply:
No no, thanks.  This really does need a comment.